.jscsrc

```diff
@@ -1,48 +1,4 @@
 {
-  "disallowKeywords": ["with"],
-  "disallowKeywordsOnNewLine": ["else"],
-  "disallowMixedSpacesAndTabs": true,
-  "disallowMultipleVarDecl": "exceptUndefined",
-  "disallowNewlineBeforeBlockStatements": true,
-  "disallowQuotedKeysInObjects": true,
-  "disallowSpaceAfterObjectKeys": true,
-  "disallowSpaceAfterPrefixUnaryOperators": true,
-  "disallowSpacesInFunction": {
-      "beforeOpeningRoundBrace": true
-  },
-  "disallowSpacesInsideParentheses": true,
-  "disallowTrailingWhitespace": true,
-  "maximumLineLength": null,
-  "requireCamelCaseOrUpperCaseIdentifiers": true,
-  "requireCapitalizedComments": true,
-  "requireCapitalizedConstructors": true,
-  "requireCurlyBraces": true,
-  "requireSpaceAfterKeywords": [
-      "if",
-      "else",
-      "for",
-      "while",
-      "do",
-      "switch",
-      "case",
-      "return",
-      "try",
-      "catch",
-      "typeof"
-  ],
-  "requireSpaceAfterLineComment": true,
-  "requireSpaceAfterBinaryOperators": true,
-  "requireSpaceBeforeBinaryOperators": true,
-  "requireSpaceBeforeBlockStatements": true,
-  "requireSpaceBeforeObjectValues": true,
-  "requireSpacesInFunction": {
-      "beforeOpeningCurlyBrace": true
-  },
-  "requireTrailingComma": {
-      "ignoreSingleLine": true
-  },
-  "requireBlocksOnNewline": true,
-  "validateIndentation": 2,
-  "validateLineBreaks": "LF",
-  "validateQuoteMarks": "'"
+  "preset": "node-style-guide",
+  "maximumLineLength": null
```

Comment:
Probably not an _easy_ way since [XO's config](https://raw.githubusercontent.com/sindresorhus/eslint-config-xo/master/index.js) is about 158 lines long.
But here's a mediocre DIFF between the current "node-style-guide" JSCS file and the defaults from XO:

```js
  rules: {
    'brace-style': [2, '1tbs', {allowSingleLine: false}],
    'camelcase': [2, {properties: 'always'}],
    'comma-dangle': [2, 'never'],
    'curly': 2,
    'indent': [2, 2, {SwitchCase: 1}], // SAME
    'key-spacing': [2, {beforeColon: false, afterColon: true}], // SAME
    'linebreak-style': [2, 'unix'], // SAME
    // MISSING: `lines-around-comment`
    'new-cap': [2, {newIsCap: true, capIsNew: true}],
    'no-mixed-spaces-and-tabs': 2, // SAME
    'no-trailing-spaces': 2, // SAME
    'no-with': 2, // SAME
    'one-var': [2, 'never'],
    'quote-props': [2, 'consistent-as-needed'],
    'quotes': [2, 'single'], // SAME
    'space-after-keywords': [2, 'always'], // SAME
    'space-before-blocks': [2, 'always'], // SAME
    'space-before-function-paren': [2, {anonymous: 'always', named: 'never'}],
    'space-in-parens': [2, 'never'], // SAME
    'space-infix-ops': 2, // SAME
    'space-return-throw-case': 2,
    'space-unary-ops': 2, // SAME
```

So about 12 of the rules are the same, and the rest may need some code tweaking.